cmd/object-api-utils.go

```diff
@@ -51,6 +51,10 @@ import (
 const (
 	// MinIO meta bucket.
 	minioMetaBucket = ".minio.sys"
+	// Background ops meta prefix
+	backgroundOpsMetaPrefix = "background-ops"
+	// MinIO Stats meta prefix.
+	minioMetaBackgroundOpsBucket = minioMetaBucket + SlashSeparator + backgroundOpsMetaPrefix
```

Comment:
minioMetaBackgroundOpsBucket - I am not able to see MakeVol of this volume, is it happening somewhere?

Reply:
> minioMetaBackgroundOpsBucket - I am not able to see MakeVol of this volume, is it happening somewhere?

it is in format-xl.go, line 774